tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Steps/Common/CommonSteps.cs

```diff
@@ -39,6 +39,13 @@ public async Task ThenTheResponseContainsTheFollowingErrors(Table table)
             actual.Should().BeEquivalentTo(expected);
         }
 
+        [Then(@"an empty object is returned")]
+        public async Task ThenAnEmptyObjectIsReturned()
+        {
+            var response = (await _response.ReadBodyAsJsonAsync());
```

Comment:
Are the brackets needed around the code await `_response.ReadBodyAsJsonAsync()`?

Reply:
👍